### ISSUE TITLE
Save player options when using repeat mode

### ISSUE THREAD

**Issue by ritiek** [mps-youtube/mps-youtube#659]
Not sure if this is possible either but at the moment one has to manually set the preferred playback speed and volume every time the track repeats itself. It would be nice if mps-youtube took care of it automatically.

**Comment by SirWojtek** [comment]
Agree. Please notice that options also return to a default value when playing a playlist. This could be frustrating when you have to volume down in each new song :wink:
